apps/web/src/components/workflow/FlowEditor.tsx

```diff
@@ -375,6 +375,7 @@ const Wrapper = styled.div<{ dark: boolean }>`
   .react-flow__attribution {
     background: transparent;
     opacity: 0.5;
+    right: 133px;
```

Comment:
this change doesn't fix the attribution problem

Reply:
@chankruze sorry for what I fixed
But would you please tell me the correct way to solve it .?
I literally want to learn new things.
And one thing more there is no tshirt in this year

Reply:
If you don't know how to fix then why you asked to be assigned and false claimed that you have already fixed those issues.

If you want to learn then fork play with it locally and silently.

I have checked your history (no contribution) but suddenly in October you decided to contribute by raising PR with shit code just to get it merged and get some swaggs.

Those Hacktobefest and Hacksquad are encouraging to learn and contribute what you are capable of. Contribution to any OSS project means understanding, interpreting and solving issue in the correct way following repo guidelines. NOT to bark "assign me assign me" having no idea of the solution.

Please, first learn then jump into this shit.

Reply:
People like you are a curse for the planet earth @chankruze

Reply:
People (uneducated noobs who try to contribute for the shake of swags) like you are curse for open source.

Reply:
@jatinder14 We maintain respectful communication in this community. Please delete/retract your comment.